dockstore-client/src/test/java/io/dockstore/client/cli/YamlValidatorTest.java

```diff
@@ -152,22 +151,6 @@ void allFilesNotPresent() {
         }
     }
 
-    @Test
```

Comment:
This case is covered [here](https://github.com/dockstore/dockstore-cli/blob/f7ba7af18ab920558a62ffc8f092a6dba4108e22/dockstore-client/src/test/java/io/dockstore/client/cli/YamlValidatorTest.java#L118), therefore this case is a duplicate, so I am deleting it.